config.json

```diff
@@ -23,11 +25,13 @@
   { "target": "ipfs/go-ipld-legacy" },
   { "target": "ipfs/go-merkledag" },
   { "target": "ipfs/go-metrics-prometheus" },
+  { "target": "ipfs/go-namesys" },
   { "target": "ipfs/go-path" },
   { "target": "ipfs/go-todocounter" },
   { "target": "ipfs/go-unixfsnode" },
   { "target": "ipfs/go-verifcid" },
   { "target": "ipfs/hang-fds" },
+  { "target": "ipfs/ipfs-update" },
```

Comment:
Unfortunately, most of the tests for this project are "sharness" (run with `cd sharness && make test`). We'll need to add a special github action to run those.

Reply:
https://github.com/ipfs/ipfs-update/pull/141